modules/gis/src/lib/geojson-to-binary.js

```diff
@@ -0,0 +1,205 @@
+    positions: new PositionDataType(pointPositions * coordLength),
+    objectIds: new Uint32Array(pointPositions)
+  };
+  const lines = {
+    pathIndices: new Uint32Array(linePaths),
+    positions: new PositionDataType(linePositions * coordLength),
+    objectIds: new Uint32Array(linePositions)
+  };
+  const polygons = {
+    polygonIndices: new Uint32Array(polygonObjects),
+    primitivePolygonIndices: new Uint32Array(polygonRings),
+    positions: new PositionDataType(polygonPositions * coordLength),
+    objectIds: new Uint32Array(polygonPositions)
+  };
+
+  const index = {
+    pointPosition: 0,
+    linePosition: 0,
+    linePath: 0,
+    polygonPosition: 0,
+    polygonObject: 0,
+    polygonRing: 0,
+    feature: 0
+  };
+
+  for (const feature of features) {
+    const geometry = feature.geometry;
+
+    switch (geometry.type) {
+      case 'Point':
+        handlePoint({coords: geometry.coordinates, points, index, coordLength});
+        break;
+      case 'MultiPoint':
+        handleMultiPoint({coords: geometry.coordinates, points, index, coordLength});
+        break;
+      case 'LineString':
+        handleLineString({coords: geometry.coordinates, lines, index, coordLength});
+        break;
+      case 'MultiLineString':
+        handleMultiLineString({coords: geometry.coordinates, lines, index, coordLength});
+        break;
+      case 'Polygon':
+        handlePolygon({coords: geometry.coordinates, polygons, index, coordLength});
+        break;
+      case 'MultiPolygon':
+        handleMultiPolygon({coords: geometry.coordinates, polygons, index, coordLength});
+        break;
+      default:
+        throw new Error('Invalid geometry type');
+    }
+
+    index.feature++;
+  };
+
+  return {
+    points,
+    lines,
+    polygons
+  };
+}
+
+function handlePoint({coords, points, index, coordLength}) {
+  points.positions.set(coords, index.pointPosition * coordLength);
+  points.objectIds[index.pointPosition] = index.feature;
+  index.pointPosition++;
+}
+
+function handleMultiPoint({coords, points, index, coordLength}) {
+  for (const point of coords) {
+    handlePoint({coords: point, points, index, coordLength});
+  }
+}
+
+function handleLineString({coords, lines, index, coordLength}) {
+  lines.pathIndices[index.linePath] = index.linePosition * coordLength;
+  index.linePath++;
+
+  lines.positions.set(coords.flat(), index.linePosition * coordLength);
+
+  const nPositions = coords.length;
+  lines.objectIds.set(new Uint32Array(nPositions).fill(index.feature), index.linePosition);
+  index.linePosition += nPositions;
+}
+
+function handleMultiLineString({coords, lines, index, coordLength}) {
+  for (const line of coords) {
+    handleLineString({coords: line, lines, index, coordLength});
+  }
+}
+
+function handlePolygon({coords, polygons, index, coordLength}) {
+  polygons.polygonIndices[index.polygonObject] = index.polygonPosition * coordLength;
+  index.polygonObject++;
+
+  for (const {} of coords) {
```

Comment:
Eslint was complaining that I didn't use the loop variable in the loop's body. Is there a better way to fix this, or is `coords.forEach` ok here?